drivers/i2c/i2c_nrfx_twi.c

```diff
@@ -211,6 +212,9 @@ static int twi_nrfx_pm_control(struct device *dev, u32_t ctrl_command,
 			case DEVICE_PM_LOW_POWER_STATE:
 			case DEVICE_PM_SUSPEND_STATE:
 			case DEVICE_PM_OFF_STATE:
+				if (pm_current_state != DEVICE_PM_ACTIVE_STATE) {
+					break;
+				}
 				nrfx_twi_uninit(&get_dev_config(dev)->twi);
```

Comment:
Why not write it:
```
if (pm_current_state != DEVICE_PM_ACTIVE_STATE) {
	nrfx_twi_uninit(&get_dev_config(dev)->twi);
}
```
wouldn't it be more readable?

Reply:
> Why not write it:
> 
> ```
> if (pm_current_state != DEVICE_PM_ACTIVE_STATE) {
> 	nrfx_twi_uninit(&get_dev_config(dev)->twi);
> }
> ```
> 
> wouldn't it be more readable?

you mean:

 ```
if (pm_current_state == DEVICE_PM_ACTIVE_STATE) {
 	nrfx_twi_uninit(&get_dev_config(dev)->twi);
}
 ```

Reply:
> you mean:
> 
> ```
> if (pm_current_state == DEVICE_PM_ACTIVE_STATE) {
> 	nrfx_twi_uninit(&get_dev_config(dev)->twi);
> }
> ```

Yes, obviously. My bad.